### ISSUE TITLE
[Keyboard Navigation - Make Code - Blocks]: There is no keyboard access for "Input Section" controls under "Blocks"

### ISSUE THREAD

**Issue by Yamini234u** [paperbits/paperbits-demo#217]
###  **User Experience:**
Users who depend on keyboard navigation will be impacted if controls are not accessible with keyboard and they will not be able to perform any desired action with the keyboard.

### **Repro Steps**

1. Open [Microsoft MakeCode for micro:bit (microbit.org)](https://makecode.microbit.org/#editor).
2.  Select the 'Blocks' option in header.
3. Now verify that user is able to insert the different category blocks in editor area or not without drag and drop feature.

### **Actual Result:**
There is no keyboard access for "Input Section" controls under "Blocks"
Observation:
While focus is on "Input" control , after pressing tab , right/left arrows and ctrl+arrows focus is not moving to the Right section of input controls.

### **Expected Result:**
 "Input Section" controls should be accessible through the keyboard.

### **MAS Reference**
[MAS 2.1.1 – Keyboard](https://aka.ms/MAS2.1.1)

### **Reference Links:**

- [Accessibility Insights!](https://accessibilityinsights.io/) - Identify accessibility bugs before check-in and make bug fixing faster and easier.
- External Bug Process: If this bug belongs to external team, mark it as resolved/done and assign it back to the tester with notes on where to file/route the bug. For more information, please use this [External Bug Process (sharepoint.com)](https://microsoft.sharepoint.com/teams/cea11y/SitePages/External-Bug-Process.aspx?xsdata=MDN8MDF8fDE1MThmM2JlZGFjMDRlZDhiZmViYTUxMDA2M2NiZWI3fDcyZjk4OGJmODZmMTQxYWY5MWFiMmQ3Y2QwMTFkYjQ3fDF8MHw2Mzc3NDEyNTM2MTEzNzIwNzR8R29vZHxWR1ZoYlhOVFpXTjFjbWwwZVZObGNuWnBZMlY4ZXlKV0lqb2lNQzR3TGpBd01EQWlMQ0pRSWpvaUlpd2lRVTRpT2lJaUxDSlhWQ0k2TVRKOQ%3D%3D&sdata=MnlpSXlVMnpGT3cvZ0xJNnFWUWZWTGt6dmo1VkJMZFpIZHBvUUduNDhyMD0%3D&ovuser=72f988bf-86f1-41af-91ab-2d7cd011db47%2cv-kgeetala%40microsoft.com). “HCL Staff should not log any third-party external bugs. Should be routed to EDAD team”.
- Please reach out to [C&AI Teams channel](https://apc01.safelinks.protection.outlook.com/?url=https://teams.microsoft.com/l/team/19%253a3cd071821b694f11b054d878453cd75f%2540thread.skype/conversations?groupId%3D30e3f968-15dd-4da2-bd8e-85f27e4af2c3%26tenantId%3D72f988bf-86f1-41af-91ab-2d7cd011db47&data=02%7c01%7cAklavyaM%40hcl.com%7c24e89c6ac26f42cfe40c08d7667e49f3%7c189de737c93a4f5a8b686f4ca9941912%7c0%7c0%7c637090564906844600&sdata=22JjBMoA1ID/KnYqFa/OB%2BrkTaLh5YYsGBd/7Bm7u5w%3D&reserved=0) for any process-related queries.

**Test Environment:**  

- OS Version: 22H2(OS Build: 22621.819)
- Browser: Microsoft Edge Version 109.0.1495.2(Official build) dev (64-bit)
- URL: [https://makecode.microbit.org/#editor](https://makecode.microbit.org/#editor)

https://user-images.githubusercontent.com/117158108/206227718-b6aa347e-9e11-4360-b726-a115d3ba7f03.mp4

**Comment by Yamini234u** [comment]
#A11yMAS , #HCL, #Accessibility, #A11ySev2, #HCL-MakeCodeArcade-Sep21,#WCAG2.1.1

**Comment by Yamini234u** [comment]
As per above comment closing this bug
